models/enterprise_sonic/static_routes/sonic_static_routes.yml

```diff
@@ -0,0 +1,94 @@
+---
+GENERATOR_VERSION: '1.0'
+ANSIBLE_METADATA: |
+    {
+        'metadata_version': '1.1',
+        'status': ['preview'],
+        'supported_by': 'community',
+        'license': 'Apache 2.0'
+    }
+NETWORK_OS: sonic
+RESOURCE: static_routes
+COPYRIGHT: Copyright 2022 Dell Inc. or its subsidiaries. All Rights Reserved
+DOCUMENTATION: |
+    module: sonic_static_routes
+    version_added: 2.0.0
+    short_description: Manage static routes configuration on SONiC
+    description:
+      - This module provides configuration management of static routes for devices running SONiC
+    author: "Shade Talabi (@stalabi1)"
+    options:
+      config:
+        type: list
+        elements: dict
+        description:
+          - Manages 'static_routes' configurations
+        suboptions:
+          vrf_name:
+            required: True
+            type: str
+            description:
+              - Name of the configured VRF on the device.
+          static_list:
+            type: list
+            elements: dict
+            description:
+              - A list of 'static_routes' configurations.
+            suboptions:
+              prefix:
+                required: True
+                type: str
+                description:
+                  - List of locally configured static routes.
+              next_hops:
+                type: list
+                elements: dict
+                description:
+                  - A list of next-hops to be utilised for the static route being specified.
+                suboptions:
+                  index:
+                    required: True
+                    type: dict
+                    description:
+                      - An identifier utilised to uniquely reference the next-hop.
+                    suboptions:
+                      blackhole:
+                        type: bool
+                        default: False
+                        description:
+                          - Indicates if route is blackholed.
```

Comment:
This description doesn't add much information.

It would be more useful to define the blackhole buzzword.

For example:

- Indicates that packets matching this route should be discarded.